pyvera/__init__.py

```diff
@@ -545,25 +545,26 @@ def set_service_value(
         self,
         service_id: Union[str, Tuple[str, ...]],
         set_name: str,
-        parameter_name: str,
-        value: Any,
+        parameter: dict,
```

Comment:
I no longer have a live vera hub - which limits my ability to check changes. As such while I can see why you want to change the parameter structure -but I'm a little reluctant because testing the refactoring needed.

Even though it's less elegant I'd be happier for you to just add an extra call for the lock api - and leave the "set_service_value" method unchanged.

Reply:
I can do that....

Reply:
Open minded.  Perhaps `call_service_with_params` or something like that?

Reply:
used set_door_code_values